pkg/cloud/providerconfig.go

```diff
@@ -91,6 +95,15 @@ func (pc *ProviderConfig) loadConfig(writeIfNotExists bool) (*CustomPricing, err
 	if !exists {
 		log.Infof("Could not find Custom Pricing file at path '%s'", pc.configFile.Path())
 		pc.customPricing = DefaultPricing()
+		// If config file is not present use the contents from mount models/ as pricing data
+		// in closed source rather than from from  DefaultPricing as first source of truth.
+		// since most images will already have a mount, to avail this facility user needs to delete the
+		// config file manually from configpath else default pricing still holds good.
+		fileName := filenameInConfigPath(pc.configFile.Path())
+		defaultPricing, err := ReturnPricingFromConfigs(fileName)
+		if err == nil {
+			pc.customPricing = defaultPricing
+		}
```

Comment:
Looks awesome, were you able to verify that this worked well with the other providers?

Reply:
Verified in GCP and Azure:

Test strategy was: 
1. use rc-6 upload a default price config and then set the new image does it show up with the same sset default - yes it did.
2. when image was set to fixed version the configs mounted on /model showed up in the /var/config as expected.

Spoke to Bolt this looks good and merging the code!